### ISSUE TITLE
Empty span before pagination buttons

### ISSUE THREAD

**Issue by amaslakov** [anubra266/choc-paginator#12]
`<span>` that's intended to hold the total number of items, when `showTotal` is set, stays even when `showTotal` is NOT set. This makes it hard to left align the pagination component.
![image](https://user-images.githubusercontent.com/10218892/143398284-7647af8d-9750-4d8c-9a2e-4e2d01000be5.png)

**Comment by anubra266** [comment]
@amaslakov I'm camping and have no access to my pc. I'll appreciate a pull request

**Comment by amaslakov** [comment]
@anubra266 Here you are #13 =)
By the way, what you think about also removing left and right margins from first and last child respectively? That should also help align pagination with page elements

**Comment by anubra266** [comment]
> @anubra266 Here you are #13 =)
> By the way, what you think about also removing left and right margins from first and last child respectively? That should also help align pagination with page elements

@amaslakov that sounds cool. Feel free to work on it.

**Comment by amaslakov** [comment]
@anubra266 I dug a bit and found that chakra's _first selector is not a `first-child`, but rather `first-of-type`, so it won't work on a mix of buttons and spans. The solution that comes to my mind is to remove first/last margins for the buttons list, and double the right margin for showTotal and left margin for page selector. What you think?

**Comment by anubra266** [comment]
> @anubra266 I dug a bit and found that chakra's _first selector is not a `first-child`, but rather `first-of-type`, so it won't work on a mix of buttons and spans. The solution that comes to my mind is to remove first/last margins for the buttons list, and double the right margin for showTotal and left margin for page selector. What you think?

Sounds cool @amaslakov

**Comment by anubra266** [comment]
> @anubra266 I dug a bit and found that chakra's _first selector is not a `first-child`, but rather `first-of-type`, so it won't work on a mix of buttons and spans. The solution that comes to my mind is to remove first/last margins for the buttons list, and double the right margin for showTotal and left margin for page selector. What you think?

Sounds cool @amaslakov

**Comment by amaslakov** [comment]
@anubra266 Okay, I updated that too. Please have a look when you have time =)
PR #13

**Comment by amaslakov** [comment]
@anubra266 Hi! Did you manage to have a look at my PR? )

**Comment by anubra266** [comment]
Yeah @amaslakov. It's good. But I can't release it without my pc. If you anyway I can do that on my iPhone. I'll be happy to.
